rbc/omniscidb.py

```diff
@@ -757,151 +897,49 @@ def is_udtf(sig):
                         sig.set_mangling('')
                     else:
                         # TODO: include device to the name mangling of UDFs
-                        if self.version < (5, 5) or not sig_is_udtf:
+                        if self.version < (5, 5):  # or not sig_is_udtf:
```

Comment:
Should the `# or not sig_is_udtf` be commented? If yes, maybe remove it.

Reply:
Fixed.